### DISCUSSION TITLE
Slint 1.7 Released with New Widgets, Multi-Window Support, and Live-Preview Redesign

### DISCUSSION THREAD

**Discussion by tronical** [slint-ui/slint#5644] [Blog Comments]
Release announcement with the highlights: https://slint.dev/blog/slint-1.7-released

[ChangeLog](https://github.com/slint-ui/slint/blob/master/CHANGELOG.md#170---2024-07-18)

<hr /><em>This discussion was created from the release <a href='https://github.com/slint-ui/slint/releases/tag/v1.7.0'>1.7.0</a>.</em>

**Comment by samrock5000** [comment]
Multi-Window Support! 🚀
This is great.

**Comment by yaobo-lab** [comment]
6666

**Comment by JakubKoralewski** [comment]
Hi, I see the "Live-Preview" section in 1.7 release docs with a nice drag-and-drop editor. Is this a private sample app or a tool we can use for WYSIWYG editing? If it's a tool then I can't find it.

**Reply by ogoffart** [reply]
This too is part of our editor extension. It is the "Slint" extension in the barcode market place or slint-lsp for other editors (https://github.com/slint-ui/slint/tree/master/editors) or also available online on https://slintpad.com 
With the extension, you need to clock on the "Show preview" code lense, or code action, to show the preview, and then use the "edit" switch at the top right to enable the mode where you can drag/drop widgets

**Reply by JakubKoralewski** [reply]
Thanks. It works through RustRover but through VSCode, despite enabling Design Mode I can see the funny crosshair cursor, but nothing apart from that (edit button, component library etc.). Anyway I've found a nice workflow with RR for now.

**Reply by ogoffart** [reply]
Very strange. Maybe the version of the extension is not up to date?

**Reply by JakubKoralewski** [reply]
It's 1.2.2

**Reply by JakubKoralewski** [reply]
Update:Mode in VSCode settings was set to "none" I was using 2 years old VSCode :D It seems to now detect latest 1.7.2 slint extension. Thanks for the tip. Works perfectly now 👍
